frontend/components/icons/Apple.tsx

```diff
@@ -1,10 +1,33 @@
 import React from "react";
 
-const Apple = () => {
+interface IAppleProps {
+  size?: "small" | "medium" | "large";
+  color?: "coreFleetBlack" | "coreVibrantBlue" | "uiFleetBlack75";
```

Comment:
if we use the values in `styles/var/colors.ts` then this type can just be `Colors` type from that file.

Reply:
Same! Built this before `Colors`. Will update and PM you.